fjord/translations/models.py

```diff
@@ -574,23 +503,29 @@ def push_translations(self):
             # than raise a GengoConfig error.
             return
 
-        balance = gengo_api.get_balance()
-        threshold = settings.GENGO_ACCOUNT_BALANCE_THRESHOLD
+        if self.gengo_watch_balance:
+            balance = gengo_api.get_balance()
+            threshold = settings.GENGO_ACCOUNT_BALANCE_THRESHOLD
```

Comment:
Testing things with gengo is difficult now and essentially entirely manual. So I was trying to reuse and centralize as much code and do it in a way that minimized code and testing required. Generating machine and human translations is identical except for this balance code which is only needed by the human translation code. I could have the human translation code override this method, do the balance check and then execute the parent method except that I need balance and threshold in the order generation loop.

So I had to choose between this and a more tangled version where I'm passing in termination functions or something like that which I thought was more complex to understand and maintain. I went with simple for now mostly because it eases the burden of testing and maintenance.